packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/PluginsAndSettings/SubCategory/index.js

```diff
@@ -136,7 +136,9 @@ const SubCategory = ({ categoryName, isFormDisabled, subCategoryName, actions, p
       </Box>
       {isModalOpen && (
         <ConditionsModal
-          headerBreadCrumbs={[categoryName, subCategoryName]}
+        headerBreadCrumbs={[
+          { id: categoryName, defaultMessage: categoryName },
+          { id: subCategoryName, defaultMessage: '' }]}
```

Comment:
thank you for the update, looks great!
could we just use `subCategoryName` as `defaultMessage` here @fhenri42? :)